src/briefcase/commands/new.py

```diff
@@ -309,7 +315,7 @@ def build_app_context(self):
         """
         formal_name = self.input_text(
             intro="""
-First, we need a formal name for your application. This is the name that will
+First, we need a formal name for your app. This is the name that will
```

Comment:
Not sure if this was intentional; but since this is a long-form explanation, we can use the full name rather than an abbreviation.